gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineConfig.java

```diff
@@ -77,6 +78,10 @@ private class BaselineUpdateConfigAction implements Action<Task> {
         private final Configuration configuration;
         private final Project rootProject;
 
+        private final String inclusiveCodeCheckOn = "inclusive-language";
+
+        private final String checkstylePathString = "checkstyle/checkstyle.xml";
```

Comment:
style nit: let's make these `private static final` constants and update to SCREAMING_SNAKE_CASE